src/scrapers/mizrahi.ts

```diff
@@ -57,8 +57,8 @@ const afterLoginSelector = '#dropdownBasic';
 const loginSpinnerSelector = 'div.ngx-overlay.loading-foreground';
 const accountDropDownItemSelector = '#AccountPicker .item';
 const pendingTrxIdentifierId = '#ctl00_ContentPlaceHolder2_panel1';
-const emailUpdatePageHebrewTitle = "עדכון כתובת דוא''ל";
-const emailUpdatePageEnglishTitle = 'Update email address';
```

Comment:
The email update page is now an embedded iframe and we no longer have an indication of it other than the url which is not really future proof.
Therefore, I added the selectors of something that will be more likely more useful (the checking account tab).